### ISSUE TITLE
Path for preformatted text

### ISSUE THREAD

**Issue by kampfflunder** [DiasPHPora/wp-to-diaspora#162]
If the Wordpress post has a `<pre>` block with text after the `</pre>`, wp-to-diaspora converts to <code>```sometext</code> (missing newline after the backticks) which is wrong.
Fix is easy:

```
# diff -u vendor/league/html-to-markdown/src/Converter/PreformattedConverter.php.org vendor/league/html-to-markdown/src/Converter/PreformattedConverter.php
--- vendor/league/html-to-markdown/src/Converter/PreformattedConverter.php.org  2018-11-29 19:23:42.703780919 +0100
+++ vendor/league/html-to-markdown/src/Converter/PreformattedConverter.php      2019-02-11 17:38:58.003479382 +0100
@@ -40,7 +40,7 @@
         $lines = preg_split('/\r\n|\r|\n/', $pre_content);
         if (count($lines) > 1) {
             // Multiple lines detected, adding three backticks and newlines
-            $markdown .= '```' . "\n" . $pre_content . "\n" . '```';
+            $markdown .= '```' . "\n" . $pre_content . "\n" . '```' . "\n";
         } else {
             // One line of code, wrapping it on one backtick.
             $markdown .= '`' . $pre_content . '`';
#
```

**Comment by noplanman** [comment]
Hi @kampfflunder, thanks for reporting this!

It's an [html-to-markdown](https://github.com/thephpleague/html-to-markdown) related issue, which seems to have been fixed in an updated version. Will update the dependencies and release a new version soon :+1:

**Comment by noplanman** [comment]
@kampfflunder I've just released version 3.0.0.

Please give it a go and give me any feedback, thanks!

**Comment by kampfflunder** [comment]
> @kampfflunder I've just released version 3.0.0.
> 
> Please give it a go and give me any feedback, thanks!

Looks good! https://pluspora.com/posts/3964585
Thank you!
